arrow-libs/optics/arrow-optics/src/commonMain/kotlin/arrow/optics/Copy.kt

```diff
@@ -0,0 +1,71 @@
+package arrow.optics
+
+public interface Copy<A> {
+  /**
+   * Changes the value of the element(s) pointed by the [Setter].
+   */
+  public infix fun <B> Setter<A, B>.set(b: B)
+
+  /**
+   * Transforms the value of the element(s) pointed by the [Traversal].
+   */
+  public infix fun <B> Traversal<A, B>.transform(f: (B) -> B)
+
+  /**
+   * Declares a block in which all optics are nested within
+   * the given [field]. Instead of:
+   *
+   * ```
```

Comment:
Should we turn these into Knit examples?
Maybe we should discuss a documentation strategy for Arrow 2.0 in the weekly meeting? 🤔

Reply:
Yep, it would be nice to have Knit examples working instead of simple blocks.